usmqe_tests/alerting/test_workload_alerting.py

```diff
@@ -0,0 +1,61 @@
+"""
+Alerting test suite - workload
+"""
+
+import pytest
+import time
+import usmqe.usmmail
+
+
+LOGGER = pytest.get_logger('workload_alerting', module=True)
+
+
+def test_cpu_alerts(workload_cpu_utilization):
+    """
+    Alerting: cpu_utilization
+    *******************
+
+    .. test_metadata:: author ebondare@redhat.com
```

Comment:
It would be good to use testcase docstring syntax seen in https://github.com/usmqe/usmqe-tests/pull/200/files and also add some `step/result` docstring.